packages/mtn-mobile/test/Adaptor.test.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+import testData from './fixtures.json' assert {type: 'json'}
+import { request } from '../src/Adaptor.js';
+
+const testServer = enableMockClient('https://fake.server.com');
+
+let state = {
+  "configuration": {
+    "baseUrl": "https://fake.server.com/",
+    "subscription_key": "fbe630ee209443d787927b08b3c57902",
+    "api_key": "9d372b835db142778a439ec2e9a2457d",
+    "api_user": "812d588a-6d74-46cd-baaf-355a2f433020"
+  }
+}
+
+describe('request', () => {
```

Comment:
One test I'd really like to see here is if the user doesn't pass an access token. That logic looks a bit more complex and I think it's valuable to validate it